guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -55,7 +55,8 @@ To use them inside an EXISTING $CAPSULE, run this command INSTEAD:
     --server-cert "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem_" \
     --server-key "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_" \
     --server-ca-cert "_/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_" \
-    --certs-update-server
+    --certs-update-server --certs-update-server-ca \
+    --certs-update-all
```

Comment:
```suggestion
    --certs-update-server \
    --certs-update-server-ca \
    --certs-update-all
```

Reply:
```suggestion
    --certs-update-all
```

This has been around for a while, but I believe has been incorrect. If `--certs-update-all` is present the other parameters are redundant. The code link below shows this:

https://github.com/theforeman/foreman-installer/blob/develop/hooks/pre/20-certs_update.rb#L28-L36

Reply:
This can be simplified to:

```
    --certs-update-all
```

Reply:
Addressed :+1: